src/main/kotlin/com/ekino/oss/recaptcha/exception/ReCaptchaValidationException.kt

```diff
@@ -0,0 +1,11 @@
+package com.ekino.oss.recaptcha.exception
+
+open class ReCaptchaValidationException(
+  val code: String,
+  override val message: String,
+  val details: List<String>? = null
+) : Exception(message)
+
+object MissingResponseException : ReCaptchaValidationException(
+  code = "recaptcha.missing.response",
+  message = "Unable to retrieve response. Please check configuration if endpoint really need reCaptcha validation or if response parameter name is correct.")
```

Comment:
I don't know if "response" term is well understood by everyone. It means the "response" String sent by the Front and produced by Google. But in our case, response could also means the payload received from Google  ReCaptcha verification API. Maybe add the term "parameter" -> "Unable to retrieve response parameter from client". What do you think ?

Reply:
Yes you're right, I updated to "Unable to retrieve recaptcha response parameter"

Reply:
Better !